### PR TITLE
Changed area search landing page texts

### PR DESCRIPTION
**Pull request by ghost**
Changed the texts and links for the area search landing page based on Abstract changes. Also added second link for the misc area search.

Relates to Taiga 1985

### CODE REVIEW

File: src/areaSearch/areaSearchLandingPage.tsx [hunk 1]
```diff
@@ -236,19 +236,33 @@ const AreaSearchLandingPage = (): JSX.Element => {
...
-                {t(
-                  'areaSearch.landingPage.otherUses.misc.blurb',
-                  'For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.',
-                )}
-              </p>
+              <div>
+                <p>
+                  {t(
+                    'areaSearch.landingPage.otherUses.misc.dogs.blurb',
+                    'For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.',
```
Comment (henrinie-nc) [inline src/areaSearch/areaSearchLandingPage.tsx:243]: To me it seems like the texts should be different, and there's two of them - when I look at the design page

File: src/areaSearch/areaSearchLandingPage.tsx [hunk 2]
```diff
@@ -236,19 +236,33 @@ const AreaSearchLandingPage = (): JSX.Element => {
...
+                <ReadMoreButton
+                  target={t(
+                    'areaSearch.landingPage.otherUses.misc.dogs.link',
+                    'https://www.hel.fi/helsinki/en/housing/plots-land-buildings/permits-for-public-areas',
+                  )}
+                />
+                <p>
+                  {t(
+                    'areaSearch.landingPage.otherUses.misc.gardening.blurb',
+                    'For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.',
```
Comment (henrinie-nc) [inline src/areaSearch/areaSearchLandingPage.tsx:255]: This is the same text as above, I would assume it should be different

File: src/areaSearch/areaSearchLandingPage.tsx [hunk 3]
```diff
@@ -236,19 +236,33 @@ const AreaSearchLandingPage = (): JSX.Element => {
...
+                  )}
+                </p>
+              </div>
```
Comment (henrinie-nc) [inline src/areaSearch/areaSearchLandingPage.tsx:258]: Missing another read me button?

Comment (henrinie-nc) [reply src/areaSearch/areaSearchLandingPage.tsx:258]: I see that it is actually below, but how come the syntax looks different compared the the one above? I can't quite grasp the nesting in this review window

File: src/i18n/en/common.json [hunk 4]
```diff
@@ -142,9 +142,15 @@
         },
         "heading": "Other land uses",
         "misc": {
-          "blurb": "For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.",
           "label": "Other land use arrangements",
-          "link": "https://www.hel.fi/helsinki/en/housing/plots-land-buildings/permits-for-public-areas"
+          "dogs": {
+            "blurb": "For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.",
```
Comment (henrinie-nc) [inline src/i18n/en/common.json:147]: The text seems different to me in the design

File: src/i18n/fi/common.json [hunk 5]
```diff
@@ -128,43 +128,49 @@
...
-          "link": "https://www.hel.fi/helsinki/fi/asuminen-ja-ymparisto/tontit/luvat/mainonta/"
+          "link": "https://www.hel.fi/fi/yritykset-ja-tyo/yritykset-ja-yrittajat/yritysten-luvat-ja-tilat"
         },
         "heading": "Muu alueenkäyttö",
         "misc": {
-          "blurb": "Esim. kaupunkiviljely, koirakoulutuskentät, mehiläistarhaus, muu maankäyttö- ja vuokraus, metsästysoikeudet , päiväkotien ulkoilualueet, vesialueen vuokraus ",
           "label": "Muut alueen\u00adkäyttö\u00adsopimukset",
-          "link": "https://www.hel.fi/helsinki/fi/asuminen-ja-ymparisto/tontit/luvat/"
+          "dogs": {
+            "blurb": "Koirankoulutuskentiltä voi vuokrata vakioviikkovuoron koirien koulutukseen.",
```
Comment (henrinie-nc) [inline src/i18n/fi/common.json:147]: These texts seem to be the correct ones, but the ones above are different - is it waiting for translation?

**Review by henrinie-nc — CHANGES_REQUESTED** [review]
Please check or clarify the difference in the text put in the elements of "Muut alueenkäyttösopimukset"

**Review by henrinie-nc — APPROVED** [review]

**Comment by ghost** [comment]
Yes, English and Swedish translations are missing. That is why there are some incorrect/duplicate texts. Due to misc land use should have 2 buttons, extra button was added to content of the BoxGridBox. Other boxes have only 1 read more button which can be found from the action section. Misc land use box has 1 read more button in the content and another in the action section.

**Pull request merged**